ui/pages/Report/components/VariantMetadata.jsx

```diff
@@ -12,7 +12,7 @@ const COLUMNS = [
   { name: 'participant_id' },
   ...VARIANT_METADATA_COLUMNS.slice(0, -1),
   { name: 'allele_balance_or_heteroplasmy_percentage' },
-  { name: 'Clinvar allele ID', format: ({ clinvar }) => clinvar?.alleleId },
+  { name: 'ClinGen allele ID', format: ({ ClinGen_allele_ID }) => ClinGen_allele_ID },
```

Comment:
we want to keep `Clinvar allele ID` here, we should just also add `ClinGen_allele_ID`

Reply:
We want to add this to the `VARIANT_METADATA_COLUMNS` so it is also shared with the individual summary data